theme_anelusia/views/snippets/s_wavy_grid.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_wavy_grid" inherit_id="website.s_wavy_grid">
+    <!-- Layout -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="data-oe-shape-data"/>
+        <attribute name="class" add="o_cc o_cc4" separator=" "/>
+    </xpath>
+    <xpath expr="//div[hasclass('o_we_shape')]" position="replace"/>
+    <xpath expr="//img" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_wavy_grid_default_image_1/web_editor/geometric/geo_square_3.svg</attribute>
+        <attribute name="data-shape">web_editor/geometric/geo_square_3</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">s_wavy_grid_default_image_1.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
+    </xpath>
+    <xpath expr="(//img)[4]" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_wavy_grid_default_image_4/web_editor/geometric/geo_square_6.svg</attribute>
+        <attribute name="data-shape">web_editor/geometric/geo_square_6</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">s_wavy_grid_default_image_4.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
+    </xpath>
+
+    <!-- Texts -->
+    <xpath expr="//h2" position="replace" mode="inner">
```

Comment:
This will have to be adapted. See community PR comment about `<span>` added by the editor ;)

Reply:
Thanks 👍  I've updated the branch and also made a little change on Bookstore, see my comment in the Community PR :)